### DISCUSSION TITLE
Execute bash command when close session/tab

### DISCUSSION THREAD

**Discussion by Grazulex** [kovidgoyal/kitty#5552] [Q&A] [answered]
Hello,

I'm starting witk Kitty and I love it.
I have create a small session config file to start with my layout, windows and app. (kitty --session otm.conf)

```
# otm.conf
new_tab Nvim
cd ~/Dev/laravel/otm
title vim
launch 'nvim'
enabled_layouts tall:bias=50;full_size=1
layout tall

new_tab Bash
cd ~/Dev/laravel/otm
launch /bin/bash 
title Bash
enabled_layouts tall:bias=50;full_size=1
layout tall

new_tab MySql
cd ~/Dev/laravel/otm
title Mysql
launch /bin/bash -c './vendor/bin/sail up -d'
launch /bin/bash -c './vendor/bin/sail mysql -h mysql -u sail -p'
enabled_layouts tall:bias=50;full_size=1
layout tall
```

I open 3 tabs, I go in the right directory and start nvim or bash commands.
Everything is fine but I need to know if I can launch a bash comman when I close a tab/session ?
I'm working with Laravel sail and I want to launch the "./vendor/bin/sail down" when I close the "mysql" tab or the session. Is it possible ?

Many Thanks.

**Comment by kovidgoyal** [comment, marked as answer]
See https://sw.kovidgoyal.net/kitty/launch/#watching-launched-windows

**Reply by Grazulex** [reply]
Many Thanks
